src/pages/guides/sellers/malware-scan.md

```diff
@@ -4,17 +4,17 @@ title: Malware scan
 
 # Malware scan
```

Comment:
Page to be reviewed by Mercury team

Reply:
@mikeweis @pdohogne-magento please review.

Reply:
This remains accurate for app submissions.

Reply:
Additionally, the app's dependencies will be automatically checked for critical security vulnerabilities (via 'npm audit'), and .